rfs_test/TEST_accountservice_schema.py

```diff
@@ -29,126 +32,217 @@
 
 
 ###################################################################################################
-# Name: Assertion_ACCO101(self, log) :Account Service                                            
-# Assertion text: 
-#The value of this property shall be a boolean indicating whether this service is enabled.  If 
-#this is set to false, the AccountService is disabled.  This means no users can be created, 
-#deleted or modified.  Any service attempting to access the Account Service, like the Session 
-#Service, will fail accessing.  Thus new sessions cannot be started with the service disabled 
-#(though established sessions may still continue operating).  Note: this does not affect Basic 
-#AUTH connections.
+# Name: Assertion_ACCO101(self, log) :Account Service
+# Assertion text:
+# The value of this property shall be a boolean indicating whether this service is enabled.  If
+# this is set to false, the AccountService is disabled.  This means no users can be created,
+# deleted or modified.  Any service attempting to access the Account Service, like the Session
+# Service, will fail accessing.  Thus new sessions cannot be started with the service disabled
+#(though established sessions may still continue operating).  Note: this does not affect Basic
+# AUTH connections.
 ###################################################################################################
 
 def Assertion_ACCO101(self, log):
 
     log.AssertionID = 'ACCO101'
-    assertion_status =  log.PASS 
+    assertion_status = log.PASS
     log.assertion_log('BEGIN_ASSERTION', None)
 
     relative_uris = self.relative_uris
-    authorization = 'on' 
+    authorization = 'on'
     rq_headers = self.request_headers()
 
-    json_payload, headers, status = self.http_GET('/redfish/v1/AccountService', rq_headers, authorization)
+    json_payload, headers, status = self.http_GET(
+        '/redfish/v1/AccountService', rq_headers, authorization)
 
     try:
         isEnabled = json_payload['ServiceEnabled']
 
         if isEnabled:
-            patch_key = 'ServiceEnabled'   
+            patch_key = 'ServiceEnabled'
             patch_value = False
-            rq_body = {patch_key : patch_value}
-            self.http_PATCH('/redfish/v1/AccountService', rq_headers, rq_body, authorization)
+            rq_body = {patch_key: patch_value}
+            self.http_PATCH('/redfish/v1/AccountService',
```

Comment:
Need to add error check here; some services won't allow a PATCH on the AccountService, which valid. If that happens, the test should PASS